### PR TITLE
update global cli flags documentation

### PR DESCRIPTION
**Pull request by fmoehler**
Related to this PR: https://github.com/cloudfoundry/bosh-cli/pull/634

### CODE REVIEW

**Comment by lnguyen** [comment]
FYI i'm waiting on PR to be merged and new CLI for this PR

**Comment by fmoehler** [comment]
https://github.com/cloudfoundry/bosh-cli/pull/634 is merged now. I guess this can be merged then as well

**Comment by fmoehler** [comment]
!! We have to update the cli version first in this PR before we merge it !!

**Review by mvach — APPROVED** [review]

**Pull request merged**
